CHANGES.rst

```diff
@@ -6,6 +6,8 @@ New Features
 
 Bug Fixes
 ^^^^^^^^^
+- Reimplementation of FluxConservingResampler to fix incorrect flux and
+  uncertainties and increase speed. [#1060]
```

Comment:
```suggestion
- Reimplementation of FluxConservingResampler. It is now faster and yields more accurate results. [#1060]
```
Rewording this to be more "user facing" - i.e., it tells what you get from this PR.